synapse/storage/database.py

```diff
@@ -600,7 +603,9 @@ async def runInteraction(
                 i.e. outside of a transaction. This is useful for transaction
                 that are only a single query. Currently only affects postgres.
```

Comment:
I think we could expand on this.

```suggestion
                that are only a single query. 
                
                Currently, this is only implemented for Postgres. SQLite will still
                run the function inside a transaction.
                
```